### PR TITLE
Test 217 test case correction

### PR DESCRIPTION
**Pull request by jaind1046**
https://drive.google.com/file/d/1G4ILh9vvLGMtoebB9VZCadU-mXN-k_k_/view?usp=sharing

### CODE REVIEW

**Comment by jaind1046** [comment]
https://drive.google.com/file/d/1tx-sN-fKomqBvnqrHj53Bp97D5WQCV-s/view?usp=sharing

File: specs/features/content-management-policy-history.feature [hunk 1]
```diff
@@ -9,10 +9,10 @@ Feature: content-management-policy-history
...
+        Given I have navigated to the Policy History page
         When Items Shown is changed to <itemCount>
-        Then up to <fileCount> previous policies are displayed
+        Then Up to <itemCount> previous policies are displayed
         Examples:
-            | itemCount | fileCount |
-            | 25        | 25        |
-            | 100       | 100       |
\ No newline at end of file
+            | itemCount |
```
Comment (NadHodeGW) [inline specs/features/content-management-policy-history.feature:16]: nothing set in the fileCount how is this checked?

**Review by NadHodeGW — CHANGES_REQUESTED** [review]
see comment in feature file

File: specs/features/content-management-policy-history.feature (hunk 1 above)
Comment (jaind1046) [reply specs/features/content-management-policy-history.feature:16]: This is not required as itemCount which we are setting will also work for the assertion. Please see the recording also.

File: specs/step_definitions/content-management-policy-history.steps.js [hunk 2]
```diff
@@ -25,15 +27,15 @@ Then('the user is taken to the Policy History page', () => {
     policyPage.assertHistoryPolicyPage()
 });
 
-Given('I have navigated to the Policy History page and there are more than {int} policies in the history', (count) => {
+Given('I have navigated to the Policy History page', () => {
     policyPage.clickOnHistoryPolicyTab()
-    policyPage.assertNumberOfRecordsOfPolicy(count)
```
Comment (NadHodeGW) [inline specs/step_definitions/content-management-policy-history.steps.js:30]: only navigation action should be inside this Given step.

File: src/pages/policy.page.js [hunk 3]
```diff
@@ -370,20 +370,22 @@ module.exports = {
...
     const element = this.options.countOfPolicies;
     I.selectOption(element, itemCount);
   },
 
-  assertPoliciesItemShownCount(itemCount) {
-    I.seeNumberOfElements(this.table.tableRows, itemCount)
+  assertPoliciesItemShownCount(itemCount, availableRecords) {
+    availableRecords.then((records) => {
+      if (records > itemCount) {
+        I.seeNumberOfElements(this.table.tableRows, itemCount)
```
Comment (NadHodeGW) [inline src/pages/policy.page.js:386]: if this records > itemCount is not the case then what happens? please revisit

**Review by NadHodeGW — CHANGES_REQUESTED** [review]
see comments in files

**Comment by jaind1046** [comment]
Test-217 after changed assersion strategy

**Review by NadHodeGW — APPROVED** [review]

**Pull request merged**